src/NuGet.Clients/NuGet.CommandLine/Common/MSBuildProjectSystem.cs

```diff
@@ -14,6 +14,8 @@ namespace NuGet.Common
     public class MSBuildProjectSystem : MSBuildUser, IMSBuildNuGetProjectSystem
     {
         private const string TargetName = "EnsureNuGetPackageBuildImports";
+        private const string JSProjectExt = ".jsproj";
+        private const string VCXProjextExt = ".vcxproj";
```

Comment:
would be `src/NuGet.Core/NuGet.ProjectManagement/Utility/Constants.cs` the right place for these constants?